### PR TITLE
Better description for required org plan for audit logs

### PR DESCRIPTION
**Pull request by ivasilov**

### CODE REVIEW

File: studio/components/interfaces/Organization/AuditLogs/AuditLogs.tsx [hunk 1]
```diff
@@ -184,7 +208,35 @@ const AuditLogs = () => {
...
+              <IconAlertTriangle className="h-4 w-4 mt-3" />
+              <div className="flex flex-row pt-3 pb-2">
+                <div className="grow">
+                  <AlertTitle_Shadcn_>
+                    Organization audit logs are not available for free or pro organizations
+                  </AlertTitle_Shadcn_>
+                  <AlertDescription_Shadcn_ className="flex flex-row justify-between gap-3">
+                    <p>
+                      If you need to use organization audit logs, then you will need to upgrade to
+                      Team or Enterprise plan to access.
```
Comment (loong) [inline studio/components/interfaces/Organization/AuditLogs/AuditLogs.tsx:225]:
```suggestion
                      Upgrade to Team or Enterprise to view up to 28 days of audit logs for your 
                      organization.
```

File: studio/components/interfaces/Organization/AuditLogs/AuditLogs.tsx [hunk 2]
```diff
@@ -184,7 +208,35 @@ const AuditLogs = () => {
...
+          {isError && error.message.endsWith('') ? (
+            <Alert_Shadcn_
+              variant="default"
+              title="Organization audit logs are not available for free or pro organizations"
+            >
+              <IconAlertTriangle className="h-4 w-4 mt-3" />
+              <div className="flex flex-row pt-3 pb-2">
+                <div className="grow">
+                  <AlertTitle_Shadcn_>
+                    Organization audit logs are not available for free or pro organizations
```
Comment (loong) [inline studio/components/interfaces/Organization/AuditLogs/AuditLogs.tsx:220]:
```suggestion
                    Organization audit logs are not available for Free or Pro organizations
```

Comment (MildTomato) [reply studio/components/interfaces/Organization/AuditLogs/AuditLogs.tsx:220]: We could just say "Free or Pro Plans" right? 

The organization detail is more for internal use.

**Review by loong — APPROVED** [review]
Copy lgtm

**Review by joshenlim — APPROVED** [review]

**Pull request merged**